Cargo.toml

```diff
@@ -29,10 +29,11 @@ oldtime = []
 wasmbind = ["wasm-bindgen", "js-sys"]
 unstable-locales = ["pure-rust-locales"]
 # Note that rkyv-16, rkyv-32, and rkyv-64 are mutually exclusive.
-rkyv-16 = ["rkyv", "rkyv?/size_16"]
-rkyv-32 = ["rkyv", "rkyv?/size_32"]
-rkyv-64 = ["rkyv", "rkyv?/size_64"]
-rkyv-validation = ["rkyv", "rkyv?/validation"]
+rkyv = ["dep:rkyv", "rkyv/size_32"]
+rkyv-16 = ["dep:rkyv", "rkyv?/size_16"]
+rkyv-32 = ["dep:rkyv", "rkyv?/size_32"]
+rkyv-64 = ["dep:rkyv", "rkyv?/size_64"]
+rkyv-validation = ["dep:rkyv", "rkyv?/validation"]
```

Comment:
So I think this one is now odd because `rkyv-validation` doesn't pick a size? I'd propose that we leave `dep:rkyv` out of `rkyv-validation` and only leave `rkyv?/validation` in there so that it will only activate the downstream feature if any of the other features have been used to enable `rkyv` (and pick a size).

Reply:
I tried both of your suggestions, but then it doesn't work out for all combinations:
```
cargo check --features=rkyv-64,rkyv-validation
    Checking rkyv v0.7.43
error: "size_32" and "size_64" are mutually-exclusive features. You may need to set `default-features = false` or compile with `--no-default-features`.
   --> C:\Users\paul\.cargo\registry\src\index.crates.io-6f17d22bba15001f\rkyv-0.7.43\src\macros.rs:127:1
    |
127 | / core::compile_error!(
128 | |     "\"size_32\" and \"size_64\" are mutually-exclusive features. You may need to set \
129 | |     `default-features = false` or compile with `--no-default-features`."
130 | | );
    | |_^

error: could not compile `rkyv` (lib) due to previous error
```

Reply:
I think we can make do without the `validation` -> `dep:rkyv` edge? It seems to work for me.

Reply:
👍 Done